presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDictionarySelectiveReader.java

```diff
@@ -594,6 +625,8 @@ private void setDictionaryBlockData(byte[] dictionaryData, int[] dictionaryOffse
             dictionaryOffsets[positionCount] = dictionaryOffsets[positionCount - 1];
             dictionaryBlock = new VariableWidthBlock(positionCount, wrappedBuffer(dictionaryData), dictionaryOffsets, Optional.of(isNullVector));
             currentDictionaryData = dictionaryData;
+            evaluationStatus = ensureCapacity(evaluationStatus, positionCount - 1);
```

Comment:
Nit: I feel the `positionCount - 1 `is confusing. Then I saw the caller passes in the +1 count:
`setDictionaryBlockData(stripeDictionaryData, stripeDictionaryOffsetVector, stripeDictionarySize + 1);`
And inside of this function we decrement 1 again.
But by convention we pass in the positionCount for the values and we know the offsets positions is the +1 count. I would just pass in the positionCount itself. But it's out of this PR so I'm fine with this now.

Reply:
-1 here because we don't have to store null for evaluation status